pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -427,6 +451,24 @@ func NewNetworkPolicyController(kubeClient clientset.Interface,
 		},
 		resyncPeriod,
 	)
+	if features.DefaultFeatureGate.Enabled(features.AdminNetworkPolicy) {
+		adminNPInformer.Informer().AddEventHandlerWithResyncPeriod(
```

Comment:
Should we check the AdminNetworkPolicy and BaselineAdminNetworkPolicy API availability before start the informer? considering the manifests are not delivered with Antrea manifests and may not be installed by users correctly before running the controller.

Reply:
Merging this with https://github.com/antrea-io/antrea/pull/5170#discussion_r1268862334